### ISSUE TITLE
syntax highlighting vscode

### ISSUE THREAD

**Issue by FloLey** [inmanta/inmanta-core#3952]

**Comment by wouterdb** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/core-5a7c152b04a1652024289b7b/issues/inmanta/inmanta-core/3952?planning-poker) with ZenHub @arnaudsjs @bartv @FloLey @Hugo-Inmanta @sanderr

**Comment by FloLey** [comment]
I think no changes are required fot inmanta-sphinx as the keywords are imported:
https://github.com/inmanta/inmanta-sphinx/blob/b95fd8660089231346011a32d96c069710017d89/sphinxcontrib/inmanta/pygments_lexer.py#L22
Am I correct?

**Comment by sanderr** [comment]
Indeed, good find. Looking at [the docs](https://docs.inmanta.com/community/dev/language.html#if-statement) confirms this as well.
